### ISSUE TITLE
Alternate Checkboxes produce indentation issues in Preview Mode

### ISSUE THREAD

**Issue by tilhil** [kepano/obsidian-minimal#430]
### Is the bug present when using the default Obsidian theme?

Yes

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

5.3.2

### Describe the bug

First of all: I love this theme, thanks for developing!

I really like the alternate chackboxes and use them frequently.
If I use the normal checkboxes `- [ ] ` and `- [x] ` and type a text spanning multiple lines, no indentation is done in preview editing mode.
If I use the alternate chackboxes e.g. `- [c] ` and type a text spanning multiple lines, the following lines are indented.

An Example in Preview Editing Mode: 

```

- Accusantium dolorem ut non illo tenetur. Maiores quia optio deserunt saepe dolore veniam. Et illo quis quibusdam maxime pariatur ipsam nesciunt saepe. Doloribus omnis et quam molestiae molestias est.
- Et qui quisquam laudantium occaecati. Ea quia odio quis porro. Enim est est eius sit voluptas quo doloremque magni. Illo natus et delectus est quae odio sequi.
- [I] Laborum quae es.
- [c] This is a faulty line and this is very annoying. I do not like that at all!
- [p] Ducimus eaque consequatur molestias  dkfj df i idf dlk flkd dignissimos perspiciatis. Eaque id numquam eveniet consequatur reiciendis doloribus voluptas qui. Aut autem quae illum et qui fuga soluta ipsam. Natus tempore eum dolore ut reiciendis aut.
- [ ] Ducimus eaque consequatur molestias  dkfj df i idf dlk flkd dignissimos perspiciatis. Eaque id numquam eveniet consequatur reiciendis doloribus voluptas qui. Aut autem quae illum et qui fuga soluta ipsam. Natus tempore eum dolore ut reiciendis aut.
- [x] Ducimus eaque consequatur molestias  dkfj df i idf dlk flkd dignissimos perspiciatis. Eaque id numquam eveniet consequatur reiciendis doloribus voluptas qui. Aut autem quae illum et qui fuga soluta ipsam. Natus tempore eum dolore ut reiciendis aut.
```

![signal-2022-09-27-140022_002](https://user-images.githubusercontent.com/114348110/192527508-1057af81-147e-42ba-9839-2f42a2d69793.png)

If I switch to reading view the indentation is gone.
If in preview editing view I insert some new lines above the list, the indentation vanishes in some cases.

The issue is present in the default obsidian theme as well, if I insert none `x` characters for "checked" checkboxes. But this case of course is not intended in that theme, since it does not have alternate checkboxes. (So I hope the bug report here isn't misplaced)

The indentation is annoying if small window widths are used. And the inconsistency (sometimes there are indents sometimes not) is iritating. I really hope you can do something about that.

love your work, thank you :)

### Debug info

Ubuntu 22.04.1 (but the issue also is present on windows and iPadOS)

SYSTEM INFO:
	Obsidian version: v0.15.9
	Installer version: v0.15.9
	Operating system: #54-Ubuntu SMP Fri Aug 26 13:26:29 UTC 2022 5.15.0-48-generic
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 14
	Plugins enabled: 2
		1: Minimal Theme Settings v5.3.2
		2: Style Settings v0.4.11

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please first try updating your theme to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by kepano** [comment]
This is already fixed in Obsidian 0.16 which you can access via the insider builds

**Comment by tilhil** [comment]
> This is already fixed in Obsidian 0.16 which you can access via the insider builds

Thanks very much. I tested it, it works and am happy. :)
